$OpenBSD: patch-src_scripttabs_cpp,v 1.1 2016/09/11 00:08:42 naddy Exp $

invalid cast from type 'std::nullptr_t' to type 'INTERP_TYPE'

--- src/scripttabs.cpp.orig	Sat May  7 19:45:17 2016
+++ src/scripttabs.cpp	Sun Sep 11 01:48:19 2016
@@ -108,7 +108,7 @@ TYPE_SYMBOL asTypeTable[] =
 	{ "",	(INTERP_TYPE)ST_POINTER_STRUCTSTAT, AT_SIMPLE, NULL, NULL },	//for NULLSTRUCTURESTAT
 
 	/* This final entry marks the end of the type list */
-	{ "END OF TYPE LIST", (INTERP_TYPE)NULL,		AT_SIMPLE,	NULL,	NULL },
+	{ "END OF TYPE LIST", (INTERP_TYPE)0,		AT_SIMPLE,	NULL,	NULL },
 };
 
 
